src/pybamm/telemetry.py

```diff
@@ -0,0 +1,36 @@
+from posthog import Posthog
+import os
+import pybamm
+import sys
+
+_posthog = Posthog(
+    # this is the public, write only API key, so it's ok to include it here
+    project_api_key="phc_bLZKBW03XjgiRhbWnPsnKPr0iw0z03fA6ZZYjxgW7ej",
+    host="https://us.i.posthog.com",
+)
+
+
+def disable():
+    _posthog.disabled = True
+
+
+_opt_out = os.getenv("PYBAMM_DISABLE_TELEMETRY", "false").lower()
```

Comment:
We should add this environment variable in the `PYBAMM_ENV` constant in `noxfile.py` as an additional measure